docbook/reference/en/en-US/modules/RESTEasy_Spring_Integration.xml

```diff
@@ -3,16 +3,21 @@
     <para>RESTEasy integrates with Springframework in various forms. In this chapter we introduce different methods to
         integrate Springframework with RESTEasy.
     </para>
+    <para>
+        <emphasis role="bold">IMPORTANT:</emphasis> As of RESTEasy 5.0.0 the Spring integration has moved to a new
+        project, group id and version. The new group id is <code>org.jboss.resteasy.spring</code>. Currently the
+        artifact id's have not changed. The current released version is 2.0.0.Alpha1.
+    </para>
     <para>RESTEasy currently supports Spring version _SPRING_VER_</para>
     <section id="BasicIntegration">
         <title>Basic Integration</title>
-        <para>For Maven users, you must use the <code>resteasy-spring</code> artifact. And here is the dependency you
+        <para>For Maven users, you must use the <code>org.jboss.resteasy.spring:resteasy-spring</code> artifact. And here is the dependency you
             should use:
         </para>
         <programlisting><![CDATA[<dependency>
-    <groupId>org.jboss.resteasy</groupId>
+    <groupId>org.jboss.resteasy.spring</groupId>
     <artifactId>resteasy-spring</artifactId>
-    <version>5.0.0-SNAPSHOT</version>
+    <version>2.0.0.Alpha1</version>
```

Comment:
Kind of related to the comments above, we might want to use the -SNAPSHOT version here, assuming at release time we replace any occurrence of the -SNAPSHOT version in the code & doc with the version to be tagged (I used to prepare for the release with a script that does all the version changes, instead of using the mvn release plugin).